test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -874,26 +984,16 @@ define([
                 if (!jobsById[jobId].batch_job) {
                     it(`processes a ${jobId} update`, async function () {
                         const jobState = jobsById[jobId];
-                        const currentState = fsmState.PROCESSING_LAUNCHED;
                         cellStartUp(this);
+                        setUpCellInState(this, 'PROCESSING_LAUNCHED');
+
                         // start up cell as if it just received the job launched message
-                        this.cell.metadata.kbase.appCell.fsm = { currentState };
                         this.cell.metadata.kbase.appCell.exec = {
                             jobState: { job_id: jobId },
                             launchState: { event: 'launched_job', job_id: jobId },
                         };
 
-                        spyOn(this.bus, 'emit').and.callFake((...args) => {
-                            const [msgType] = args;
-                            if (msgType === jcm.MESSAGE_TYPE.START_UPDATE) {
-                                // send a status update
-                                TestUtil.send_STATUS({
-                                    bus: this.bus,
-                                    jobId,
-                                    jobState,
-                                });
-                            }
-                        });
```

Comment:
Since there is no longer a `START_UPDATE` message, the status message is sent manually below.